assets/sass/components/_issue-sidebar.scss

```diff
@@ -282,29 +273,34 @@
 
   .i-bar-item.is-active {
     background: rgba(var(--c-#{$status}-rgb), 0.05);
-  }
-
-  .i-bar-item.is-#{$status} {
-    --check-color: rgba(var(--c-#{$status}-rgb), 0.8);
-
-    &:hover,
-    &:focus {
-      background: rgba(var(--c-#{$status}-rgb), 0.05);
-    }
-
-    .i-bar-check-initial {
-      background: var(--check-color);
+    strong {
+      font-weight: bold;
+      color: var(--c-dark)
     }
+    
+  }
 
-    .i-bar-check-completed,
-    .i-bar-check-percent {
-      border: 2px solid var(--check-color);
-
-      i {
-        color: var(--check-color);
+    .i-bar-item.is-#{$status} {
+      --check-color: rgba(var(--c-#{$status}-rgb), 0.8);
+  
+      &:hover,
+      &:focus {
+        background: rgba(var(--c-#{$status}-rgb), 0.05);
+      }
+  
+      .i-bar-check-initial {
+        background: var(--c-success);
```

Comment:
the diff for this section is confusing because of my reformatting, sorry, i basically just changed line 292